edgelet/docker-rs/src/models/host_config.rs

```diff
@@ -184,8 +184,8 @@ pub struct HostConfig {
     // #[serde(rename = "DnsSearch", skip_serializing_if = "Option::is_none")]
     // dns_search: Option<Vec<String>>,
     // /// A list of hostnames/IP mappings to add to the container's `/etc/hosts` file. Specified in the form `[\"hostname:IP\"]`.
-    // #[serde(rename = "ExtraHosts", skip_serializing_if = "Option::is_none")]
-    // extra_hosts: Option<Vec<String>>,
+    #[serde(rename = "ExtraHosts", skip_serializing_if = "Option::is_none")]
```

Comment:
What scenario does this enable? I believe specifying extrahosts in createOptions was already working?

Reply:
The scenario is the following:
We read the extra host give to the agent
We run the diagnostic module with the "extrahost" parameter and we resolve the hostname.

To be able to read extra host we need to enable this.